aztec/src/main/kotlin/org/wordpress/aztec/watchers/FullWidthImageElementWatcher.kt

```diff
@@ -30,18 +32,19 @@ class FullWidthImageElementWatcher(val aztecText: AztecText) : TextWatcher {
     }
 
     private fun normalizeEditingAroundImageSpans(count: Int, start: Int) {
-        if (!aztecText.isTextChangedListenerDisabled()) {
+        if (aztecText.text.getSpans(0, 0, FullWidthImageProcessingMarker::class.java).isEmpty()) {
```

Comment:
Thanks for the comment. Can you please explain why it's better to also check for `isTextChangedListenerDisabled()`? Thanks!